### PR TITLE
Add Google Command

### LINKED ISSUE #78: Google command
Creates a https://letmegooglethat.com link with the search query. Syntax would be `+google <query>`. Good memes.

### PR DESCRIPTION
**Pull request by JohnnyJTH**
Resolves #78

### CODE REVIEW

File: carberretta/bot/cogs/links.py [hunk 1]
```diff
@@ -59,6 +59,13 @@ async def command_pep(self, ctx: commands.Context, pep_number: int) -> None:
                 return
 
             await ctx.send(f"PEP {pep_number:04}: <https://python.org/dev/peps/pep-{pep_number:04}>")
+            
+    @commands.command(name="google", aliases=['lmgt', 'lmgtfy'])
+    async def command_google(self, ctx, *, query):
```
Comment (parafoxia) [inline carberretta/bot/cogs/links.py:64]:
```suggestion
    async def command_google(self, ctx: commands.Context, *, query: str) -> None:
```

File: carberretta/bot/cogs/links.py [hunk 2]
```diff
@@ -59,6 +59,13 @@ async def command_pep(self, ctx: commands.Context, pep_number: int) -> None:
...
+        if len(query) > 500:
+            return await ctx.send("Please keep the query under 500 characters")
```
Comment (parafoxia) [inline carberretta/bot/cogs/links.py:66]:
```suggestion
            return await ctx.send("Your query should be no longer than 500 characters.")
```

**Review by parafoxia — APPROVED** [review]
A few minor changes, but I can handle them on this end. Good work!

**Review by parafoxia — DISMISSED** [review]

**Review by parafoxia — CHANGES_REQUESTED** [review]
Remove the meme command from this PR.

**Comment by JohnnyJTH** [comment]
> Remove the meme command from this PR.

I tried but I can't. I'll just make a new PR because on my current fork the meme command is gone but this wont update

**Comment by JohnnyJTH** [comment]
https://github.com/Carberra/Carberretta/pull/83

**Comment by parafoxia** [comment]
> I tried but I can't. I'll just make a new PR because on my current fork the meme command is gone but this wont update

#80 was closed already. You should not make a new PR with a meme command.

**Pull request closed without merging**